libr/bin/format/elf/elf.c

```diff
@@ -1250,6 +1253,9 @@ static ut64 get_import_addr(ELFOBJ *bin, int sym) {
 			case EM_ARM:
 			case EM_AARCH64:
 				plt_addr = Elf_(r_bin_elf_get_section_addr) (bin, ".plt");
+				if (plt_addr == 0) {
+					plt_addr = Elf_(r_bin_elf_get_section_addr_end) (bin, ".rela.plt");
```

Comment:
Not always correct

Reply:
？？？？？？？？don't understand to use `.rela.plt`

Reply:
I think phdr and shdr should be two separate views.

If you want to use phdr, use `DT_JMPREL` (offset) `DT_PLTRELSZ` (size) `DT_PLTREL` (element size).
`DT_REL` and `DT_RELA`

`musl/ldso/dynlink.c`